os_reststack_manager/config/cloud_config.yml.sample

```diff
@@ -0,0 +1,4 @@
+extra_files:
```

Comment:
The three yaml files in the config directory are now duplicated, they have a .yml.sample and a .yml version. I don't think this is wise, the service should fail if it hasn't been configured by the admin, that is why the .yml file were missing, to force the person installing to configure the app. I think the .yml files should be removed.